R/BiocWorkshopSubmit.R

```diff
@@ -62,12 +62,78 @@ BiocWorkshopSubmit <- function(...) {
+                        id = "form",
+                        textInput(
+                            "id",
+                            mandatory("Workshop ID"),
+                            placeholder = "abc123"
+                        ),
+                        textInput(
+                            "title",
+                            label = mandatory("Title"),
+                            placeholder = "A Bioconductor Workshop Title"
+                        ),
+                        textInput(
+                            "section",
+                            label = mandatory("Section"),
+                            placeholder = "BioC2023"
+                        ),
+                        ## TODO: point out workshop.bioconductor.org examples
+                        textInput("description", "Description"),
+                        textInput(
+                            "ghrepo",
+                            "GitHub Repository",
+                            placeholder = "username/repository"
+                        ),
+                        textInput(
+                            "startfile", "Start File", value = "README.md"
+                        ),
+                        textInput(
+                            "url",
+                            "Container URL",
+                            placeholder = "ghcr.io/username/repo"
+                        ),
+                        textInput("tag", "Container Tag", placeholder = "latest"),
+                        textInput("pkglist", "List of packages to pre-install", placeholder="S4Vectors,username/repo,GenomicRanges"),
+                        textInput("vignettes", "List of vignettes to add to container", placeholder="'vignettes/workshop.Rmd,vignettes/workshop2.Rmd' in source repository OR a url list eg: 'https://gist.githubusercontent.com/example/20823a9e7123cc/raw/1a8ec84131286a47926237089de6/workshop.Rmd,https://raw.githubusercontent.com/example/myworkshop/devel/vignettes/workshop2.Rmd'"),
+                        actionButton("render", "Render", class = "btn-primary")
+                    ),
+                    hidden(
+                        div(
+                            id = "render_msg",
+                            h3("Review the GitHub issue comment on the right"),
+                            actionButton(
+                                "post", "Create Issue", icon("paper-plane"),
+                                style = .ISSUE_BTN_CSS,
+                                class = "btn-danger"
+                            )
+                        )
+                    ),
+                    hidden(
+                        div(
+                            id = "thankyou_msg",
+                            h3("Submitted successfully!")
+                        )
+                    ),
+                    hidden(
+                        span(id = "submit_msg", "Submitting..."),
+                        div(id = "error",
```

Comment:
@almahmoud This block is indented differently and it looks like some lines have changed but others haven't.

Reply:
Yeh that was an annoying oversight on my part when I fixed the conflicts in the browser. I fixed it in the latest commit, so the diff of the PR now actually shows the little bit that was added/changed